pkg/cmd/server/origin/master_config.go

```diff
@@ -236,8 +244,18 @@ func BuildMasterConfig(options configapi.MasterConfig) (*MasterConfig, error) {
 	internalkubeInformerFactory := kinternalinformers.NewSharedInformerFactory(privilegedLoopbackKubeClientsetInternal, defaultInformerResyncPeriod)
 	externalkubeInformerFactory := kinformers.NewSharedInformerFactory(privilegedLoopbackKubeClientsetExternal, defaultInformerResyncPeriod)
 	authorizationInformers := authorizationinformer.NewSharedInformerFactory(authorizationClient, defaultInformerResyncPeriod)
+	templateInformers := templateinformer.NewSharedInformerFactory(templateClient, defaultInformerResyncPeriod)
 	informerFactory := shared.NewInformerFactory(internalkubeInformerFactory, externalkubeInformerFactory, privilegedLoopbackKubeClientsetInternal, privilegedLoopbackOpenShiftClient, customListerWatchers, defaultInformerResyncPeriod)
 
+	err = templateInformers.Template().InternalVersion().Templates().Informer().AddIndexers(cache.Indexers{
```

Comment:
This causes the template informer to always be started, even if the template broker is off.  This needs to be guarded.